src/main/java/com/iexec/worker/compute/post/PostComputeService.java

```diff
@@ -63,6 +63,17 @@ public PostComputeService(
         this.computeExitCauseService = computeExitCauseService;
     }
 
+    /**
+     * This method implements the post-compute part of the workflow dedicated to standard tasks.
+     * <p>
+     * This method implements almost the same algorithm thant the one executed for TEE tasks in
+     * a tee-worker-post-compute container.
+     * Classes names are inlined in comments for comparison.
+     * @param taskDescription description of a standard task
+     * @return a post compute response with a cause in case of error. The response is returned to
+     *         {@link com.iexec.worker.compute.ComputeManagerService}
+     * @see com.iexec.worker.compute.ComputeManagerService
```

Comment:
```suggestion
     * This method implements the post-compute part of the workflow dedicated to standard tasks,
     * following almost the same algorithm than the one executed for TEE tasks in
     * a tee-worker-post-compute container.
     * Classes names are inlined in comments for comparison.
     * @param taskDescription description of a standard task
     * @return a post compute response with a cause in case of error. The response is returned to
     *         {@link com.iexec.worker.compute.ComputeManagerService}
     * @see com.iexec.worker.compute.ComputeManagerService
```
?

Reply:
Comment improved a bit in 1f19396
I kept a concise first sentence to keep a summary and rephrased the following to avoid having basically the same text.